github.com/kanban/projects.user.js

```diff
@@ -1,23 +1,22 @@
 // ==UserScript==
 // @name        GH Kanban (projects)
-// @namespace   github.javiertorres.eu
+// @namespace   github.saleiva.eu
 // @include     https://github.com/orgs/*/projects/*
 // @include     https://github.com/*/*/projects/*
 // @version     1.0.2
 // @grant       GM_getValue
 // @grant       GM_setValue
 // @require     https://code.jquery.com/jquery-3.1.1.min.js
-// @updateURL   https://raw.githubusercontent.com/javitonino/grease/master/github.com/kanban/projects.user.js
+// @updateURL   https://github.com/saleiva/grease/raw/master/github.com/kanban/projects.user.js
 // ==/UserScript==
 
-var ISSUE_TAG_STYLE = 'margin-right: 5px; border: 1px solid rgba(0, 0, 0, 0.1); border-radius: 3px; font-size: 12px; padding: 2px 5px;';
 var ISSUE_REFERENCES_CACHE = {};
 var ISSUE_DATA_CACHE = {};
 var TOKEN = GM_getValue('oauth_token');
 var USER_LOGIN = $('meta[name=user-login]').attr('content');
 var IGNORED_COLUMNS = ['Done'];
 
-function getIssueTimeline(card_link, callback) {
+function getIssueData(card_link, callback) {
```

Comment:
When reading the code, I thought it was clearer to understand if timeline refers to milestones and data to PR. But now I see what you say. Whatever you think is better!